fixieai/agents/code_shot.py

```diff
@@ -30,6 +30,13 @@
 _AGENT_ID_JWT_CLAIM = "aid"
 
 
+@pydantic_dataclasses.dataclass
+class ModelSettings:
```

Comment:
Model is a over-used terminology, I think `LlmSettings` might deliver the meaning quicker and to the point